packages/connect-form/src/components/advanced-options-tabs/tls-ssl-tab/tls-ssl-tab.tsx

```diff
@@ -0,0 +1,125 @@
+import { css } from '@emotion/css';
+import React, { useCallback } from 'react';
+import {
+  Description,
+  Label,
+  RadioBox,
+  RadioBoxGroup,
+  spacing,
+} from '@mongodb-js/compass-components';
+import ConnectionStringUrl from 'mongodb-connection-string-url';
+
+import { UpdateConnectionFormField } from '../../../hooks/use-connect-form';
+import FormFieldContainer from '../../form-field-container';
+import { TLS_OPTIONS } from '../../../constants/ssl-tls-options';
+
+const descriptionStyles = css({
+  marginTop: spacing[1],
+});
+
+const TLS_TYPES = [
+  {
+    value: TLS_OPTIONS.DEFAULT,
+    label: 'Default',
+    description:
+      'Default (unset) - TLS/SSL will be used when connecting using a DNS Seed List Connection Format (mongodb+srv://) and it will not be used when connecting using a standard connection string schema format (mongodb://).',
+  },
+  {
+    value: TLS_OPTIONS.ON,
+    label: 'On',
+    description: 'On - TLS/SSL is enabled for this connection.',
+  },
+  {
+    value: TLS_OPTIONS.OFF,
+    label: 'Off',
+    description: 'Off - TLS/SSL is disabled for this connection.',
```

Comment:
actually, one small nit, because – does look a bit nicer than - :slightly_smiling_face: 

```suggestion
      'Default (unset) – TLS/SSL will be used when connecting using a DNS Seed List Connection Format (mongodb+srv://) and it will not be used when connecting using a standard connection string schema format (mongodb://).',
  },
  {
    value: TLS_OPTIONS.ON,
    label: 'On',
    description: 'On – TLS/SSL is enabled for this connection.',
  },
  {
    value: TLS_OPTIONS.OFF,
    label: 'Off',
    description: 'Off – TLS/SSL is disabled for this connection.',
```